pkg/cli/start.go

```diff
@@ -22,7 +22,7 @@ var startCmd = &cobra.Command{
 	Start the telemetry server, which will start listening for data on
 	the input device specified via the --source flags
 	`,
-	Example: `  telemetry start -source=/dev/tty.* [--port=port]`,
+	Example: `  telemetry start -source=/dev/tty.* [--port=port] [--fake] [--tty=/dev/...]`,
```

Comment:
What's the difference between `--source` and `--tty`? Also, I think your example is missing a `-` in `-source=`.

Reply:
Oops, that was previously from Karl's variant of this which he never implemented. Fixed.